scripts/2-build-index.sh

```diff
@@ -10,3 +10,4 @@ INDEX_FILE_NAME="index.json"
 echo "Building index...";
 $SCRIPTS_DIR/"build-index.py" $UNIVERSE_DIR;
 echo "OK";
+
```

Comment:
This script does not appear to be up to date. Can we double check that all scripts are up to date?